src/components/PrivateRoute/PrivateRoute.jsx

```diff
@@ -0,0 +1,11 @@
+import { Navigate, Outlet } from "react-router-dom";
+import { useAuth } from "../../context/auth-context";
+
+export const PrivateRoute = () => {
```

Comment:
This component can be refactored to be more reusable and making it a wrapper component.

Look at the example in this blog: https://dev.to/iamandrewluca/private-route-in-react-router-v6-lg5

If you have any doubt feel free to ping me on discord.

Reply:
Thanks @shobit1337 will read about it and implement it.